src/dispatch.jl

```diff
@@ -123,7 +123,136 @@ function plot_electric_dispatch(d::Dict; title="Electric Systems Dispatch", save
         push!(traces, PlotlyJS.scatter(
             name = "ElectricStorage SOC",
             x = 1:T,
-            y = d["ElectricStorage"]["year_one_soc_series_pct"],
+            y = d["ElectricStorage"]["year_one_soc_series_fraction"],
+            yaxis="y2",
+            marker = PlotlyJS.attr(
+                color="rgb(100,100,100)",
+            ),
+        ))
+
+        layout = PlotlyJS.Layout(
+            title_text = title,
+            xaxis_title_text = "time step",
+            yaxis_title_text = "kW",
+            yaxis2 = PlotlyJS.attr(
+                title = "SOC",
+                overlaying = "y",
+                side = "right"
+            )
+        )
+    end
+    p = PlotlyJS.plot(traces, layout)
+
+    if save_html
+        PlotlyJS.savefig(p, replace(title, " " => "_") * ".html")
+    end
+
+    PlotlyJS.plot(traces, layout)  # will not produce plot in a loop
+
+end
+
+
+
+function plot_mpc_dispatch(d::Dict; title="MPC Dispatch", save_html=false)
+
+    traces = PlotlyJS.GenericTrace[]
+
+    layout = PlotlyJS.Layout(
+        title_text = title,
+        xaxis_title_text = "time step",
+        yaxis_title_text = "kW"
+    )
+    
+    eload = d["ElectricLoad"]["load_series_kw"]
```

Comment:
Currently there is no ElectricLoad in the outputs from MPC. One could add the ElectricLoad from MPC request into the results dictionary but then we would need to change `load_series_kw` to `loads_kw`.

Similarly, there is no ElectricStorage -> to_load_kw_series. I can add these two in MPC outputs so we can get the plot to work.

Reply:
Nevermind, i tested with mpc-multinode branch and it works!

Reply:
Yeah I originally tied this package to a version of REopt.jl but REopt is so big and this package is so tiny it didn't seem worth requiring one to download REopt to use REoptPlots. Do you approve? Should I merge?